### PR TITLE
Related to issue #39 .FIXED bug for exec install_dev_db.sh

### PR DESCRIPTION
**Pull request by ertugertshkulaku**
## Pull Request description
I FIXED the part of failed to execute script install_dev_db.sh

## General checklist

- [ ] You have reviewed the guidelines [document](https://docs.google.com/document/d/1wqcOafLx5hth8SAg4dqV_LV3un3m5WYR8RdTJ4MbbUM/edit?usp=sharing).
- [ ] PR name contains fix version. For example, "[3.3.4] Hotfix of some UI component" or "[3.4] New Super Feature".
- [ ] The [milestone](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/about-milestones) is specified and corresponds to fix version.  
- [ ] Description references specific [issue](https://github.com/thingsboard/thingsboard/issues).
- [ ] Description contains human-readable scope of changes.
- [ ] Description contains brief notes about what needs to be added to the documentation.
- [ ] No merge conflicts, commented blocks of code, code formatting issues.
- [ ] Changes are backward compatible or upgrade script is provided.
- [ ] Similar PR is opened for PE version to simplify merge. Required for internal contributors only.
  
## Front-End feature checklist

- [ ] Screenshots with affected component(s) are added. The best option is to provide 2 screens: before and after changes;
- [ ] If you change the widget or other API, ensure it is backward-compatible or upgrade script is present.
- [ ] Ensure new API is documented [here](https://github.com/thingsboard/thingsboard-ui-help)

## Back-End feature checklist

- [ ] Added corresponding unit and/or integration test(s). Provide written explanation in the PR description if you have failed to add tests.
- [ ] If new dependency was added: the dependency tree is checked for conflicts.
- [ ] If new service was added: the service is marked with corresponding @TbCoreComponent, @TbRuleEngineComponent, @TbTransportComponent, etc.
- [ ] If new REST API was added: the RestClient.java was updated, issue for [Python REST client](https://github.com/thingsboard/thingsboard-python-rest-client) is created.

### CODE REVIEW

**Comment by volodymyr-babak** [comment]
Hello @ertugertshkulaku 

Thanks for contributing into this project - that's awesome!

File: application/src/main/resources/tb-edge.yml [hunk 1]
```diff
@@ -494,6 +494,9 @@ spring:
     open-in-view: "${SPRING_JPA_OPEN_IN_VIEW:false}"
     hibernate:
       ddl-auto: "none"
+      naming:
```
Comment (volodymyr-babak) [inline application/src/main/resources/tb-edge.yml:497]: @ertugertshkulaku 
Could you please provide more details on these changes below? Why do you think they are necessary?

Comment (ertugertshkulaku) [reply application/src/main/resources/tb-edge.yml:497]: I encountered an error in creating the database and after changing this configuration I solved it, but I understand that it works without this configuration.  I will make a new commit where I have the tb_edge.yml in the initial state.

**Comment by volodymyr-babak** [comment]
@ertugertshkulaku thanks for this finding and pull request!

**Pull request merged**